droidlet/tools/crowdsourcing/sync_whitelists.py

```diff
@@ -102,8 +113,37 @@ def add_workers_to_quals(add_list: list, qual: str):
         # Check to make sure the qualification was added successfully
         if not worker.is_qualified(qual):
             logging.info(f"!!! {worker} not successfully qualified, debug")
+            add_all_success = False
         else:
             logging.info(f"Worker {worker.worker_name} added to list {qual}")
+    return add_all_success
+
+
+def revoke_worker_qual(turker: str, qual: str):
+    """
+    revoke worker qualification,
+    return if revoke succedded
+    """
+    try:
+        db_id = db.new_worker(turker, "mturk")
+        worker = Worker.get(db, db_id)
+    except:
+        worker = db.find_workers(turker, "mturk")[0]
+
+    try:
+        db.make_qualification(qual)
+    except:
+        pass
+    else:
+        logging.debug(f"{qual} qualification not exists, so create one")
```

Comment:
We shouldn't be creating a qualification just to revoke a worker from them, if that's even possible.  If the qualification doesn't exist, log a message to the user and quit.